ffi/freetype_h.lua

```diff
@@ -2,6 +2,7 @@ local ffi = require("ffi")
 
 ffi.cdef[[
 typedef int FT_Error;
+typedef long int FT_Pos;
```

Comment:
Did ffi-cdecl handle everything okay, or did you have to manually post-process some stuff here?

Reply:
It did. No stdout/stderr output when running it that way (with full paths stripped out):
```
CPPFLAGS=
CPPFLAGS+=-I./base/thirdparty/freetype2/build/git_checkout/freetype2/include
CPPFLAGS+=' '
CPPFLAGS+=-I./base/thirdparty/harfbuzz/build/git_checkout/harfbuzz/src
CPPFLAGS+=' '
CPPFLAGS+=-I./base/thirdparty/fribidi/build/git_checkout/fribidi/lib
export CPPFLAGS
./ffi-cdecl/ffi-cdecl gcc ./base/ffi-cdecl/freetype2_decl.c ./base/ffi/freetype_h.lua
```
and everything worked with the results.
(May be uneeded `-I` stuff, I had some experiments with these other libs before going with a Lua module.)

Reply:
Cool, thanks!